.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+defaults: &defaults
+  working_directory: /routemaster
+  docker:
+    - image: marcinw/circle:latest
```

Comment:
I know, it's not perfect. If I had access to `deliveroo` organization on Docker Hub, I can put it there. Essentially that's a simple Docker-in-Docker image with a bunch of [useful dependencies](https://github.com/marcinwyszynski/circle/blob/master/Dockerfile).